packages/ui/src/form-elements/map/index.tsx

```diff
@@ -75,6 +76,11 @@ const MapField: FC<MapProps> = ({
         center = calculateCenter(polygon);
     }
 
+    const zoom = {
+        minZoom: props?.map?.minZoom ? parseInt(props.map.minZoom) : 7,
+        maxZoom: props?.map?.maxZoom ? parseInt(props.map.maxZoom) : 20
```

Comment:
Hier ook doorvoeren graag, dat kan dan 1 functie worden. Misschien ook in admin panel al afvangen? Dat maxZoom dus nooit lager kan zijn dan minZoom.